core/vm/interface.go

```diff
@@ -104,16 +102,3 @@ type StateDB interface {
 	// Finalise must be invoked at the end of a transaction
 	Finalise(bool)
 }
-
-// CallContext provides a basic interface for the EVM calling conventions. The EVM
-// depends on this context being implemented for doing subcalls and initialising new EVM contracts.
-type CallContext interface {
```

Comment:
Searching for `vm.CallContext` across github yields only https://github.com/Fantom-foundation/go-ethereum-substate/blob/ab1d0339012b5f67dae1c8bcff7ff1429feeff36/core/vm/evm.go#L133 which looks like an old version of the source. Probably safe to remove.